cartographer/utils/version.py

```diff
@@ -7,9 +7,15 @@ class JSONAPIVersion(Enum):
     JSONAPI_1_0 = "1.0"
 
 
-JSONAPI_VALID_VERSIONS = [JSONAPIVersion.JSONAPI_RC2, JSONAPIVersion.JSONAPI_RC3, JSONAPIVersion.JSONAPI_1_0]
-JSONAPI_DEFAULT_VERSION = JSONAPIVersion.JSONAPI_1_0
+JSONAPI_VALID_VERSIONS = [version for version in JSONAPIVersion]
+_JSONAPI_DEFAULT_VERSION = JSONAPIVersion.JSONAPI_1_0
 
 
 def get_default_version():
-    return JSONAPI_DEFAULT_VERSION
+    global _JSONAPI_DEFAULT_VERSION
```

Comment:
I think `global` is necessary only in the setter, at least according to this article:

http://stackoverflow.com/questions/423379/using-global-variables-in-a-function-other-than-the-one-that-created-them

Ignore this if you've already tested this and found it not to be true.